MarkupsToModel/MRML/vtkMRMLMarkupsToModelNode.cxx

```diff
@@ -198,9 +201,9 @@ void vtkMRMLMarkupsToModelNode::ReadXMLAttributes( const char** atts )
       nameString << attValue;
       nameString >> this->KochanekTension;
     }
-
   }
-  //vtkDebugMacro("XML atts number of tools read "<<GetNumberOfTools());
+
+  this->Modified();
```

Comment:
Instead of hardcoding calling of Modified(), use Set... functions, which take care of calling Modified() if something is actually modified. Call StartModify/EndModify to prevent multiple Modified event invokations (see for example how it is done in vtkMRMLSegmentEditorNode).